external.bzl

```diff
@@ -668,6 +668,19 @@ def _kythe_contributions():
         remote = "https://github.com/kythe/lang-proto",
     )
 
+def _extractor_image_dependencies():
+    """Defines external repositories necessary for extractor images."""
+    git_repository(
```

Comment:
generally prefer http_archive of a github .zip file, rather than git_repository.  The former will download the snapshot of that version, the latter will download the entire commit history (and then largely discard it).

Reply:
Done for bazelisk. Is there a corresponding "archive" for go_repositories?

Reply:
NAFAIK